src/components/login/LoginForm/PasswordInput/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { passwordInputStyle, errorTextStyle } from "./style";
+
+export default function PasswordInput({
+  onValidityChange,
+}: {
+  onValidityChange: (valid: boolean) => void;
+}) {
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  const validatePassword = (value: string) => {
+    if (!value) return "PW를 입력해주세요.";
+    if (value.length < 8) return "PW는 최소 8글자 이상이어야 합니다.";
+    return "";
+  };
+
+  const handleBlur = () => {
+    const message = validatePassword(password);
+    setError(message);
+    onValidityChange(!message);
+  };
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    setPassword(value);
+    const isValid = validatePassword(value) === "";
+    if (error) setError("");
+    onValidityChange(isValid);
```

Comment:
이런 로직이라면 사용자가 입력 중일 때 onBlur로 발생한 메시지가 지워지는데요! UX 상에서 조금 어색해보여요. (타이핑 도중에도 에러 메시지가 실시간으로 바뀌는게 부드러움)
저라면 아래와 같이 썼을 것 같아요. `EmailInput`도 같은 방식으로 수정해주시면 좋을 것 같아요!
```suggestion
    const message = validatePassword(value);
    setError(message);
    onValidityChange(message === "");
```

Reply:
요구사항을 다시 확인해보니 "Input Field를 벗어났을 때"라는 조건이 있었군요!
이 피드백 무시해주셔도 좋지만, 개인적으로는 반영해주셔도 좋을 것 같아요. (데모 페이지에서는 타이핑 도중에도 에러 메시지가 실시간으로 바뀌더라고요.)